### ISSUE TITLE
On-screen lyrics do not appear

### ISSUE THREAD

**Issue by GoogleCodeExporter** [memeco/osd-lyrics#100]
```
What steps will reproduce the problem?
1. Start Amarok
2. Start playing music
3. Start OSD Lyrics
4. Notice that the tray icon does appear, but nothing else

What's the output in terminal?
$ osd-lyrics 
(osd-lyrics:6947): Gdk-CRITICAL **: gdk_window_is_visible: assertion
`GDK_IS_WINDOW (window)' failed
*Error*: in function ol_osd_window_paint: ol_osd_window.c[1319]
assert GTK_WIDGET_REALIZED (widget) failed

(osd-lyrics:6947): Gdk-CRITICAL **: gdk_window_is_visible: assertion
`GDK_IS_WINDOW (window)' failed
*Error*: in function ol_osd_window_paint: ol_osd_window.c[1319]
assert GTK_WIDGET_REALIZED (widget) failed

(... printed again and again a lot of times...)

What's the player and which version you are using?
Amarok 2.3.1

What's version of OSD Lyrics?
0.3.20100604

What's your operating system?
Kubuntu 10.04 64-bit with KDE 4.4.4
```

Original issue reported on code.google.com by `windypo...@gmail.com` on 4 Jun 2010 at 5:56

**Comment by GoogleCodeExporter** [comment]
```
It's probably because the OSD Window is hidden.
Right click the tray icon, check whether the `Hide' item is checked.
```

Original comment by `tigerso...@gmail.com` on 10 Jun 2010 at 4:12

**Comment by GoogleCodeExporter** [comment]
```
Hahaha... That is quite embarrassing indeed.

I did notice something else while the lyrics were displayed. The lyrics seem to 
occupy a rectangle near the bottom-left corner of the screen. I can click 
through them. However, the rectangle starting from the top left corner of the 
screen (0,0) and with the same size as the area the lyrics take, is 
unclickable. Moreover, if I put my mouse out of this unclickable area, then 
mouse my mouse back over it, then right when the mouse enters the unclickable 
area, the lyrics take a gray background and some controls appear below them, 
but the background and the controls disappear instantly and the lyrics look the 
same again.
```

Original comment by `windypo...@gmail.com` on 10 Jun 2010 at 6:14

**Comment by GoogleCodeExporter** [comment]
```
Would you please attach a screenshot for that?
```

Original comment by `tigerso...@gmail.com` on 11 Jun 2010 at 1:45

**Comment by GoogleCodeExporter** [comment]
```
Here goes. I noticed that I can drag the area and it will move the lyrics with 
it.

In fact, it seems that the "area" occupied by the lyrics is this area, but the 
actual lyrics are displayed with a big offset from it.
```

Original comment by `windypo...@gmail.com` on 11 Jun 2010 at 2:14

Attachments:
- [osdlyrics.png](https://storage.googleapis.com/google-code-attachments/osd-lyrics/issue-100/comment-4/osdlyrics.png)

**Comment by GoogleCodeExporter** [comment]
```
It's an invisible window to detect mouse event. It should be at the same place 
as the lyric window. This might be a bug or incompatible with the window 
manager of KDE. I guess if you check `Lock position' in the popup menu, the 
invisible window will disappear. 
```

Original comment by `tigerso...@gmail.com` on 11 Jun 2010 at 4:00

**Comment by GoogleCodeExporter** [comment]
```
Yes, it does indeed disappear. Also, if I check "Lock position", then mouse 
over over the lyrics (the real lyrics, not the area in the top left), they 
become half-transparent. Thus I think the lyric window handles mouseover events 
fine. Why use a secondary, invisible window for detecting mouse events then?
```

Original comment by `windypo...@gmail.com` on 11 Jun 2010 at 4:48

**Comment by GoogleCodeExporter** [comment]
```
i am not able to see osd lyrics with kde (kubuntu)
```

Original comment by `piyush.w...@gmail.com` on 24 Jun 2010 at 4:00

**Comment by GoogleCodeExporter** [comment]
```
The problem in KDE should be fixed in 0.4.0
```

Original comment by `tigerso...@gmail.com` on 15 Jun 2011 at 12:18
- Changed state: **Fixed**
